### DISCUSSION TITLE
Server Down

### DISCUSSION THREAD

**Discussion by mediasaranainformatika** [kirimwa-id/forum#139] [Tanya Jawab]
Halo mas rio, terima kasih utk produk kirimwa api nya.

Mau tanya mas, kenapa setver nya sering down ya? Sampai saat ini kami sering putus koneksi dengan server, 503 error nya. Terima kasih mas

**Comment by rioastamal** [comment]
Halo @mediasaranainformatika,

Pengguna sudah mencapai 1300+ dan ribuan pesan dikirim per harinya. Sepertinya ada memory leaks di server dengan pengguna sebanyak itu. Kedepan saya berencana melakukan optimasi.

Opsi lain adalah versi enterprise dengan dedicated instance (tidak shared dengan user lain) dan unlimited message. Opsi ini belum saya luncurkan di publik namun jika memang tertarik opsi berbayar ini silahkan kontak saya pribadi.

Thanks
Rio

**Comment by mediasaranainformatika** [comment]
Terima kasih mas rio,

Kalo boleh tau utk versi berbayar, biaya brpa ya mas?

**Reply by rioastamal** [reply]
Silahkan kirim email ke rio at kirimwa.id
